.github/workflows/flake8-checker.yml

```diff
@@ -23,6 +23,12 @@ jobs:
         run: |
           pip install -r repo-info/requirements.txt
 
+      - name: Install Flake8
+        run: |
+          pip install flake8
+
       - name: Run Flake8
         run: |
           flake8
+
```

Comment:
Please remove these unnecessary changes.

Reply:
moved to PR #781